account_payment_partner/models/account_payment_method.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountPaymentMethod(models.Model):
+    _inherit = 'account.payment.method'
+
+    show_bank_account = fields.Selection(
+        [
+            ('full', 'Full'),
+            ('first', 'First n chars'),
+            ('last', 'Last n chars'),
+            ('no', 'No'),
+        ],
+        string='Show bank account',
+        default='full',
+        help="Show in invoices partial or full bank account number")
+    show_bank_account_chars = fields.Integer(
+        string="Number of chars",
```

Comment:
For the sake of helping to know which field is each one when using advanced search, I would put here a more descriptive `string`: `Nº of digits for customer bank account`, and a label with only `# of digits`